cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -407,39 +218,32 @@ class ClusterHttpManagement(
 
       val bootstrapSettings = ClusterBootstrapSettings(bootstrapConfig)
       val bootstrapContactPointRoutes = new HttpClusterBootstrapRoutes(bootstrapSettings)
+      ClusterBootstrap(system).setSelfContactPoint(Uri(s"http://$hostname:$port"))
```

Comment:
This won't work when the port is 'ephemeral' (0/auto-assigned by the OS), right? and in case of NAT?